docs/resources/Guild.md

```diff
@@ -890,13 +892,14 @@ Modify a guild role. Requires the `MANAGE_ROLES` permission. Returns the updated
 
 ###### JSON Params
 
-| Field       | Type    | Description                                                    |
-| ----------- | ------- | -------------------------------------------------------------- |
-| name        | string  | name of the role                                               |
-| permissions | string  | bitwise value of the enabled/disabled permissions              |
-| color       | integer | RGB color value                                                |
-| hoist       | boolean | whether the role should be displayed separately in the sidebar |
-| mentionable | boolean | whether the role should be mentionable                         |
+| Field       | Type                                     | Description                                                         |
+| ----------- | ------- | ---------------------------------------------------------------------------------------------------- |
+| name        | string  | name of the role                                                                                     |
+| permissions | string  | bitwise value of the enabled/disabled permissions                                                    |
+| color       | integer | RGB color value                                                                                      |
+| hoist       | boolean | whether the role should be displayed separately in the sidebar                                       |
+| icon        | string  | the role's [icon hash](#DOCS_REFERENCE/image-formatting) (if the guild has the `ROLE_ICONS` feature) |
```

Comment:
When updating a role via the PATCH, `icon` is a file (specifically ImageType)

Reply:
How do I write that?

Reply:
Update the table here https://github.com/discord/discord-api-docs/blob/master/docs/resources/Guild.md#modify-guild-role--patch-guildsguildiddocs_resources_guildguild-objectrolesroleiddocs_topics_permissionsrole-object to add `icon` and mention it is a file type, which requires multipart/form-data body

Reply:
Will do later

Reply:
the client sends a data uri in json ([image data](https://discord.com/developers/docs/reference#image-data) in the docs), not multipart/form-data

Reply:
So am I right?